File: BlackJack/src/shared/models/Statistic/ResponseGetGameStatisticView.ts
import { PlayerStatisticView } from './PlayerStatisticView';

import { ResponseWinnerGameView } from '../Game/ResponseWinnerGameView';

export class ResponseGetGameStatisticView {
    gameId: string;
    bots: Array<PlayerStatisticView>;
    user: PlayerStatisticView;
    cardsleft: number;
    winner: ResponseWinnerGameView;
}

File: BlackJack.ViewModels/Statistic/ResponseGetGameStatisticView.cs
using System.Collections.Generic;

namespace BlackJackViewModels.Statistic
{
    public class ResponseGetGameStatisticView
    {
        public string GameId { get; set; }
        public List<PlayerStatisticView> Bots { get; set; }
        public PlayerStatisticView User { get; set; }
        public string Winner { get; set; }

        public ResponseGetGameStatisticView()
        {
            Bots = new List<PlayerStatisticView>();
            User = new PlayerStatisticView();
        }

    }
}
